src/components/graph/menus/modification-list-item.js

```diff
@@ -103,22 +110,52 @@ export const ModificationListItem = ({
         [modif, handleToggle]
     );
 
+    const getBranchStatusModificationValues = (modification) => {
+        return {
+            action: modification?.action,
+            energizedEnd: modification?.energizedVoltageLevelId,
+            computedLabel: <strong>{modification?.equipmentId}</strong>,
+        };
+    };
+
+    const getEquipmentAttributeModificationValues = (modification) => {
+        return {
+            equipmentAttributeName: modification?.equipmentAttributeName,
+            equipmentAttributeValue: modification?.equipmentAttributeValue,
+            computedLabel: <strong>{modification?.equipmentId}</strong>,
+        };
+    };
+
     useEffect(() => {
         if (!studyUuid || !currentNode || !modif) {
             return;
         }
-        setComputedValues({
-            energizedEnd: modif.energizedVoltageLevelId,
-            computedLabel: <strong>{getComputedLabel()}</strong>,
-        });
+        const modificationValues = JSON.parse(modif.messageValues);
+
+        switch (modif.messageType) {
+            case MODIFICATION_TYPES.BRANCH_STATUS_MODIFICATION.type:
+                setComputedValues(
+                    getBranchStatusModificationValues(modificationValues)
+                );
+                break;
+            case MODIFICATION_TYPES.EQUIPMENT_ATTRIBUTE_MODIFICATION.type:
```

Comment:
If i'm not mistaken this case wasn't handled before, what does it fixes or add ?

Reply:
it's specific case that we should adapt after modification done on the back end